research-software-directory/RSD-as-a-service
# Migrate releases

# Migrate releases

Changes proposed in this pull request:

* migration of releases
* change FIXME's to null for projects

How to test:
After building and running, you should see content on the following pages:
* http://localhost:3500/release
* http://localhost:3500/release_content
* http://localhost:3500/release?select=*,release_content(doi,tag)

On the page http://localhost:3500/project you should *not* be able to find the string `FIXME`.

PR Checklist:
